requirements/edx/base.txt

```diff
@@ -690,6 +702,8 @@ packaging==21.0
     # via
     #   bleach
     #   drf-yasg
+pact-python==1.4.0
+    # via edxval
```

Comment:
Shouldn't this be a testing only dependency?  If so, I think we should fix this in edxval so we don't have to pull in all these new packages here.